requests_futures/sessions.py

```diff
@@ -19,7 +19,7 @@
     print(response.content)
 
 """
-from concurrent.futures import ThreadPoolExecutor, ProcessPoolExecutor
+from concurrent.futures import Future, ThreadPoolExecutor, ProcessPoolExecutor
```

Comment:
Does `Future` need to be imported here? Would be surprised if it doesn't cause a lint warning with pyflakes.

Reply:
My bad, I initially used it but it caused Python 2.7 incompatibilities.
I removed it from the code and forgot to remove the import.

Will do it right now